spec/api/commentable_spec.rb

```diff
@@ -125,11 +125,11 @@ def test_unicode_data(text)
       subject { post "/api/v1/#{commentable_id}/threads", parameters }
 
       shared_examples_for 'CommentThread creation API' do |context='course'|
-        it 'creates a new CommentThread' do
+        it 'creates a new CommentThread and marks it as read for owner user' do
           expect(CommentThread.count).to eq 0
 
           body = parse(subject.body)
-          expect(body).to include('read' => false,
+          expect(body).to include('read' => true,
```

Comment:
Doesn't this already check that the thread is read?  Why do we need the call to `test_thread_marked_as_read`?  If there is a way to avoid the new call to `test_thread_marked_as_read` altogether, including the above tests, and always just check thread read is true, it seems like the test would be simpler and more accurate, in case anything else changes about the implementation of how the read status gets computed.